changelog.dd

```diff
@@ -358,6 +362,56 @@ assert(a[0] == 42);
 -------
 )
 
+$(LI $(LNAME2 inner-class, `std.traits.isInnerClass` was added to identify
+    Java-style nested classes with an accessible `outer` pointer)
+    $(P Classes that are nested inside other classes (and not inside functions)
+    and that don't define any `outer` symbol have an `outer` field which allows
```

Comment:
I would use commas here as it might be easier for the readers to keep the context: "Classes, that .. and don't define any `outer` symbol, have an ..."